docs/src/components/Layout/Page/Page.component.tsx

```diff
@@ -96,35 +89,55 @@ const StyledHeader = styled('header')`
     }
 `;
 
+const SectionNavCol = styled(Col)`
+    padding-left: 0;
+    padding-right: 0;
+`;
+
 interface StyledSectionNavProps {
-    breakpoint?: BreakpointsType;
+    sectionNavOpen: boolean;
 }
 
 const StyledSectionNav = styled('div')<StyledSectionNavProps>`
-    box-sizing: border-box;
     background-color: background.base;
     border-right: light;
-    ${({ breakpoint }) => responsiveCSS(breakpoint, [
-        breakpoints.xs,
-        breakpoints.sm,
-    ])`
-        position: fixed;
-        width: 100vw;
-        height: 120%;
-        z-index: 90;
-        overflow-y: scroll;
-        top: 4.5rem;
-        left: 0;
-        right: 0;
-        padding-bottom: 30rem;
-    `}
+    box-sizing: border-box;
+    height: 120%;
+    left: 0;
+    overflow-y: scroll;
+    padding-bottom: 30rem;
+    position: fixed;
+    right: 0;
+    top: 4.5rem;
+    width: 100vw;
+    z-index: 90;
+    display: ${({ sectionNavOpen }) => (sectionNavOpen ? 'display' : 'none')}
```

Comment:
Has this been linted? Curious about the indentation. Also, if we're setting the `display` prop value we should use the xstyled-comp sytax: `${(props) => css({ display: props ? 'block' : 'none' })}` or ```${(props) => css`display: ${props ? 'block' : 'none'}```` instead of `display: ${(props) => props ? 'block' : 'none'`

Reply:
pretend quotes are backticks.

Reply:
Yep, this is post-lint. What's the `css` module give us in this case? I don't have an issue with using it, but unclear as to the why

Reply:
We aren't overriding a string, per se, we're overriding a css object. IIRC xstyled prefers this and it won't throw a warning in our TS compiler about type mismatches.